docs/pages/versions/unversioned/sdk/screen-capture.mdx

```diff
@@ -128,6 +128,46 @@ export default function useScreenCaptureCallback() {
 
 </SnackInline>
 
+## Screen Recording Events
+
+Screen recording detection is available on iOS 11 and newer. The API lets you detect when users start or stop recording your app's screen, allowing you to protect sensitive content.
+
+#### Example
```

Comment:
```suggestion
```

No need for this since its a common pattern to put an example which is followed by some context.